### DISCUSSION TITLE
How do we do scaling

### DISCUSSION THREAD

**Discussion by dennissetiawan** [indeedeng/iwf#266] [Q&A]
If we see increasing latency for workflow to finish, what  do we need to do so that the iwf can scale?
Do we need to configure / fine tune some hard things on the worker  like scaling when scaling temporal?

Do you consider iwf as production ready?
Please provide a thorough guide to do this :)

**Comment by longquanzheng** [comment]
We are using it production so yes, it’s production ready. 

I will add more instructions later. But basically, for server, it depends on which backed you use, cadence or temporal, just scale /operate it as a typical cadence/temporal application. 

For IWF application, it’s just a rest service so it’s very simple to scale like any other rest service

**Comment by longquanzheng** [comment]
See https://github.com/indeedeng/iwf/wiki/iWF-Application-Operations and https://github.com/indeedeng/iwf/wiki/iWF-Server-Operations
